index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header><nav>
+        <a href="#" > Gul</a>
+        <a href="#" ><img src="images/Union.png"></a></nav>
+    </header>
+
+    <section id="top">
+        <h1> I’m Umarah Gul. Glad to see you!
+        </h1>
+        <p class="pg1">I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t
+            hesitate to contact me.</p>
+        <h2 class="connect">LET'S CONNECT</h2>
+        <ul class="footer">
+            <li><a href="https://www.twitter.com"><img src="images/twitter.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/github.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/Linkedin icon.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/medium.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/angellist.png" /></a></li>
+        </ul>
+    </section>
```

Comment:
I noticed that you're using fixed widths and heights for several elements. As per the project requirements, it's important to make the website responsive across different screen sizes. Therefore, I suggest avoiding fixed widths and heights and instead using responsive techniques like percentage-based or flexible units to ensure the elements adapt well to various screen sizes.
![image](https://github.com/umarahgul/Portfolio/assets/55840999/c8130340-f882-46c9-883f-6418d75ac61d)